agenta-backend/agenta_backend/models/shared_models.py

```diff
@@ -65,3 +65,10 @@ class HumanEvaluationScenarioOutput(BaseModel):
 class TemplateType(enum.Enum):
     IMAGE = "image"
     ZIP = "zip"
+
+
+class AppType(enum.Enum):
```

Comment:
I'm torn by this one to be honest.
- I fail to see how this is a named attribute instead of just a tag -- extra maintainability rigidity
- Application Type as defined here seems ill-defined to me -- extra maintainability ambiguity

It's not `CUSTOM` vs `TEMPLATE`
It's not `HOSTED` vs `HOOKED`
It's not `AGENT` vs `CHAIN` vs `TASK` vs `EMBEDDING` vs `RETRIEVAL` vs `GENERATION` vs `CONVERSATION`

It's a mix.

Reply:
Got it.